a2/g_kevin_k_biju.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
```

Comment:
Name the file with capital letters next time i.e. `a2/G_kevin_k_biju.c` instead of `a2/g_kevin_k_biju.c`. Merging PR for now.